app/Http/Controllers/ShiftController.php

```diff
@@ -530,6 +537,16 @@ private function updateStatus($shift) {
         return ["status"=>"fa fa-question", "style"=>"color:lightgrey;", "title"=>"Dienst frei"];
     }
 
+    /**
+     * @param $shift
+     * @return bool
+     */
+    public function isAllowedToChange($shift): bool
+    {
+        $clubEvent = $shift->schedule->event;
+        $createPersonLdapId = $clubEvent->creator ? $clubEvent->creator->person->prsn_ldap_id : null;
+        return \Auth::hasUser() && \Auth::user()->hasPermissionsInSection($clubEvent->section, RoleUtility::PRIVILEGE_CL) || Person::isCurrent($createPersonLdapId);
```

Comment:
maybe add parenthesis to make the logical statement more clear (I am not sure about operator precedence for `$a && $b || $c`).